csrankings-a.csv

```diff
@@ -1055,7 +1055,7 @@ Amit Chakrabarti,Dartmouth College,http://www.cs.dartmouth.edu/~ac,LjP106QAAAAJ
 Amit Chattopadhyay,IIIT Bangalore,http://amitchattopadhyay.weebly.com,sooauB0AAAAJ
 Amit Daniely,Hebrew University of Jerusalem,http://amitdaniely.com,jUtYwE0AAAAJ
 Amit Dua,BITS Pilani,http://www.bits-pilani.ac.in/pilani/amitdua/profile,NOSCHOLARPAGE
-Amit Dvir,Ariel University,http://www.ariel.ac.il/sites/amitd,S6wdYwIAAAAJ
+Amit Dvir, Ariel University, https://www.ariel.ac.il/wp/amitd/, fYOS6GcAAAAJ

```

Comment:
Please remove spaces after commas.

Reply:
Hi, 

Thanks a lot. 

I moved the space after the comma